Traceback (most recent call last):
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\asyncio\base_events.py", line 664, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\OKTAVIAN\AppData\Roaming\Python\Python312\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
no = int(input('Masukkan berita ke berapa yang ingin di cari kata kuncinya : '))
kalimat = nltk.sent_tokenize(df['isi_baru'][no])
kalimat = [sentence.replace('.','') for sentence in kalimat]
kata = nltk.word_tokenize(df['isi_baru'][no])
kata = list(set(key for key in kata if key != '.'))

kalimat
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [1;32mIn[4], line 1[0m
[1;32m----> 1[0m no [38;5;241m=[39m [38;5;28mint[39m([38;5;28;43minput[39;49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mMasukkan berita ke berapa yang ingin di cari kata kuncinya : [39;49m[38;5;124;43m'[39;49m[43m)[49m)
[0;32m      2[0m kalimat [38;5;241m=[39m nltk[38;5;241m.[39msent_tokenize(df[[38;5;124m'[39m[38;5;124misi_baru[39m[38;5;124m'[39m][no])
[0;32m      3[0m kalimat [38;5;241m=[39m [sentence[38;5;241m.[39mreplace([38;5;124m'[39m[38;5;124m.[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124m'[39m) [38;5;28;01mfor[39;00m sentence [38;5;129;01min[39;00m kalimat]

File [1;32m~\AppData\Roaming\Python\Python312\site-packages\ipykernel\kernelbase.py:1281[0m, in [0;36mKernel.raw_input[1;34m(self, prompt)[0m
[0;32m   1279[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[0;32m   1280[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[1;32m-> 1281[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[0;32m   1282[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[0;32m   1283[0m     [38;5;28mstr[39m(prompt),
[0;32m   1284[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[0;32m   1285[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[0;32m   1286[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[0;32m   1287[0m )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

